<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Idesk-x-set</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_Exit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>430</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>601</width>
      <height>421</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Иконки</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>321</width>
        <height>381</height>
       </rect>
      </property>
      <property name="title">
       <string>Параметры иконки</string>
      </property>
      <widget class="QLineEdit" name="Idesk_Caption">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Название иконки</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Idesk_CaptionTip">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Название типа</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Idesk_Command">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Исполняемая команда</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Idesk_Icon">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Картинка для иконки</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Command">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>150</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Обзор</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Icon">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>200</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Обзор</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_prev">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>60</y>
         <width>48</width>
         <height>48</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>301</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string>Размер иконки на рабочем столе</string>
       </property>
       <widget class="QSpinBox" name="Idesk_Width">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>48</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="Idesk_Height">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>30</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>48</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>30</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Ширина</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>30</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Высота</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>300</y>
         <width>301</width>
         <height>71</height>
        </rect>
       </property>
       <property name="title">
        <string>Координаты на рабочем столе</string>
       </property>
       <widget class="QSpinBox" name="Idesk_X">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
        <property name="maximum">
         <number>1024</number>
        </property>
        <property name="value">
         <number>780</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="Idesk_Y">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>30</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="maximum">
         <number>600</number>
        </property>
        <property name="value">
         <number>140</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>30</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>X</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>30</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>251</width>
        <height>381</height>
       </rect>
      </property>
      <property name="title">
       <string>Список иконок</string>
      </property>
      <widget class="QComboBox" name="List_icon">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_edit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Редактировать</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_del">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>90</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>231</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Выбрать существующую иконку</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Desktop">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Выбрать desktop-файл</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Заполнить из desktop-файла</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Save">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>340</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Сохранить и перезапустить Idesk</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Общие настройки</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
